examples/deadline/All-In-AWS-Infrastructure-Basic/python/package/lib/service_tier.py

```diff
@@ -117,11 +118,13 @@ def __init__(self, scope: Construct, stack_id: str, *, props: ServiceTierProps,
             stage=Stage.from_directory(props.docker_recipes_stage_path)
         )
 
+        version_query = VersionQuery(self, 'VersionQuery')
```

Comment:
I'm not sure how to handle this. If you pip install from PyPI (which is how the example is set up to work), this won't work because VersionQuery hasn't been released yet. If you package locally and pip install the result of that, it will work.

If we remove this now to keep the example working, it will eventually break when we do our next release, and we'll need to remember to come back and update it then.

Reply:
You should be able to `pip install` the self-built Python package that contains your change into the example.

Reply:
Yes there's no problem doing that, I mentioned it works in my original comment. The issue is what should our expectations be on the customer for them to get the example to work. I guess recommending they only work off the release branch should be fine. I'll update the readme to state that and also add step-by-step instructions on how to get it to work with the latest mainline version by running `pack.sh` and then `pip install` for the packaged python version.